Mixpanel/MPVariant.m

```diff
@@ -137,6 +141,7 @@ - (id)initWithCoder:(NSCoder *)aDecoder
         self.experimentID = [(NSNumber *)[aDecoder decodeObjectForKey:@"experimentID"] unsignedLongValue];
         self.actions = [aDecoder decodeObjectForKey:@"actions"];
         self.tweaks = [aDecoder decodeObjectForKey:@"tweaks"];
+        self.targeting = [aDecoder decodeBoolForKey:@"targeting"];
```

Comment:
I thought it wouldn't but actually this is a valid concern. If an app saves an archive and then gets updated this code will be executed on the old archive which won't have the targeting property.